shared/src/data/GoodNumber.ts

```diff
@@ -216,20 +216,24 @@ export class GN {
     return new GN(this.int.neg().toFixed(0), this.resolution, this.base);
   }
 
+  /**
+   * Recklessly multiplies the `GN` by a `BigSource` without checking for precision loss.
```

Comment:
I'm not sure the "without checking for precision loss" is relevant here. The note below about non-integers is sufficient.